dev/com.ibm.ws.jsf.2.2/bnd.bnd

```diff
@@ -132,14 +132,13 @@ Import-Package: \
   !javax.ejb, \
   !javax.persistence, \
   !org.apache.myfaces.ee6, \
-  !net.sf.cglib.proxy, \
   *
 
 Include-Resource: \
   META-INF=@src/META-INF, \
   @${repo;com.ibm.websphere.javaee.jsf.2.2}!/META-INF/resources/**, \
   @${repo;com.ibm.websphere.javaee.jsf.2.2}!/META-INF/internal-resources/**, \
-  @${repo;commons-digester:commons-digester;2.1}, \
+  @${repo;commons-digester:commons-digester;2.1}!/META-INF/**, \
```

Comment:
I think `!/META-INF/**, \`  should be `!/!META-INF/**, \` ( second exclamation mark) 

```
-includeresource: @${repo;com.acme.foo;latest}!/!META-INF/* | Extract the contents of the highest found com.acme.foo version  in the bnd workspace repository into the root folder of the target JAR,  preserving relative paths, excluding the META-INF/ folder
````
https://bnd.bndtools.org/instructions/includeresource.html

Last build didn't contain any digester classes within the JSP bundles.